source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
Shouldn't that be `init(...arguments) {`? I thought `arguments` the keyword was "gone" from ES2015.
Thanks for the clarification :)